src/screens/Game/TimeContainer/index.tsx

```diff
@@ -37,15 +37,16 @@ function TimeContainer({game}: Props) {
     timeStyle: 'short'
   };
   const firstPlayed = new Date(tsInfo.firstPlayed)
-  const firstDate = new Intl.DateTimeFormat(storage.getItem('language') || 'en', options).format(firstPlayed);
+  const language = storage.getItem('language')?.replace('_', '-').toLowerCase()
```

Comment:
Is pt-br expected here? Shouldn't it be pt-BR?

Reply:
Hmm, actually both work.